src/main/java/tests/FileReaderWriterTest.java

```diff
@@ -0,0 +1,29 @@
+package tests;
+
+import core.basesyntax.FileReader;
+import core.basesyntax.FileReaderImpl;
+import core.basesyntax.FileWriter;
+import core.basesyntax.FileWriterImpl;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class FileReaderWriterTest {
+    @Test
+    public void testFileReaderWriter() throws IOException {
+        String filePath = "testReport.csv";
```

Comment:
Avoid using hardcoded file paths in test methods. It's better to define them as constants or pass them as parameters to make the tests more flexible and maintainable.